RealmSwift/SwiftUI.swift

```diff
@@ -176,10 +176,24 @@ private final class ObservableStoragePublisher<ObjectType>: Publisher where Obje
     private var subscribers = [AnySubscriber<Void, Never>]()
     private let value: ObjectType
     private let keyPaths: [String]?
+    private let unwrappedValue: () -> (ObjectBase?)
```

Comment:
Why is this a closure and not just a value?

Reply:
it should be a value